pulser-core/pulser/json/abstract_repr/schemas/noise-schema.json

```diff
@@ -103,6 +103,9 @@
         },
         "temperature": {
           "type": "number"
+        },
+        "with_leakage": {
+          "type": "boolean"
```

Comment:
I guess this is not needed, we can just check if `"leakage"` is in the `noise_types`. That way we won't need to make any modifications to the schema